docs/source/usage.rst

```diff
@@ -524,28 +523,33 @@ Custom interpolations can also receive the following special parameters:
 - ``_parent_`` : the parent node of an interpolation.
 - ``_root_``: The config root.
 
-This can be achieved by adding ``_parent_`` and ``_root_`` to the resolver signature.
+This can be achieved by adding the special parameters to the resolver signature.
 Note that special parameters must be defined as named keywords (after the `*`):
 
+In this example, we use ``_parent_`` to implement a sum function that defaults to 0 if the node does not exist.
+(In contrast to the sum we defined earlier where accessing an invalid key, e.g. `"a_plus_z": ${sum:${a}, ${z}}` will result in an error).
 
 .. doctest::
 
     >>> OmegaConf.register_new_resolver(
-    ...     "sum_friends", 
-    ...     lambda a, b, *, _parent_: _parent_[a] + _parent_[b], 
-    ...     use_cache=False
+    ...     "sum2",
+    ...     lambda a, b, *, _parent_: _parent_.get(a, 0) + _parent_.get(b, 0),
```

Comment:
Unfortunately this line just slightly too long to fit in the doc code box.
Suggestion:
```python
    >>> def sum2(a, b, *, _parent_):
    ...     return _parent_.get(a, 0) + _parent_.get(b, 0)
    >>> OmegaConf.register_new_resolver(
    ...     "sum2",
    ...     sum2,
    ...     use_cache=False,
    ... )
```
(when I switch the default value of `use_cache` to False it will look better)

Reply:
Good catch. did you check out the PR and built the doc to check?

Reply:
Yes -- this got me a few times before so it caught my eye